CONTRIBUTING.md

```diff
@@ -64,46 +78,18 @@ First, make sure you have the prerequisites installed and available on your `$PA
 
-Next, get the code:
+### Building our repository
 
 1. Fork this repo
 2. Clone your fork locally (`git clone https://github.com/<youruser>/azure-sdk-for-js.git`)
 3. Open a terminal and move into your local copy (`cd azure-sdk-for-js`)
 4. Install and link all dependencies (`rush update`)
+5. Build the code base (`rush build`)
 
-### Making the switch
-
-If you have previously worked in this repo using the `npm` workflow, the first time you switch to using Rush you should commit or stash any untracked files and then get back to a clean state by running `rush reset-workspace` before proceeding any further. This will get rid of any latent package-lock files, as well as your existing (incompatible) node_modules directories. You can then proceed down the path outlined below.
-
-### Using Visual Studio Code
-
-#### Debugging
-
-Debugging Node.js code in VSCode is [well documented](https://code.visualstudio.com/docs/nodejs/nodejs-debugging). However, starting from version 1.45.1, VSCode can automatically debug Node.js code in most cases without having to write custom `launch.json` files for that purpose and this is true for our SDKs code as well. A demonstration of that feature can be found in the [release notes](https://code.visualstudio.com/updates/v1_45#_automatic-debug-configurations).
-
-#### Warnings
-
-Visual Studio Code has a feature which will automatically fetch and install @types packages for you, using the standard npm package manager. This will cause problems with your node_modules directory, since Rush uses PNPM which lays out this directory quite differently. It's highly recommended that you ensure "Typescript: Disable Automatic Type Acquisition" is checked in your VSCode Workspace Settings (or ensure `typescript.disableAutomaticTypeAcquisition` is present in your .vscode/settings.json file).
-
-The current version of VSCode for Windows has a bug that may cause a "file locked" error when you run any Rush command that modifies your node_modules directory:
-
-```
-ERROR: Error: Error: EPERM: operation not permitted, mkdir 'C:\XXXXX\node_modules'
-Often this is caused by a file lock from a process such as your text editor, command prompt, or "gulp serve"
-```
-
-This bug is fixed in the Insiders build of VSCode (1.34), and will be included in the next release. Until then, you can resolve this by running the "Typescript: Restart TS server" command from the Command Palette to release the lock on the files.
-
-### Warnings for Windows users
-
-Git for Windows has a bug where repository files may be unintentionally removed by `git clean -df` when a directory is locally linked. Because Rush creates local links between packages, you may encounter this. It's highly recommended to use the `rush reset-workspace` command to get your working directory back to a clean state instead. If you prefer to run `git clean -df` manually, you must first run `rush unlink` so that the operation can be performed safely.
```

Comment:
I don't see `rush reset-workspace` documented anymore - can we preserve that part somewhere? It seems like a useful command to fix problems.

Reply:
it is mentioned in the `Getting back to a clean state` section here.

Reply:
thanks for pointing that out - somehow Edge's "find in page" missed that